plugin.xml

```diff
@@ -40,7 +40,7 @@
 
         <framework src="src/android/build.gradle" custom="true" type="gradleReference" />
         <source-file src="src/android/HockeyApp.java" target-dir="src/com/zengularity/cordova/hockeyapp" />
-        <lib-file src="src/android/HockeySDK-Android/HockeySDK-5.0.4.aar" />
+        <lib-file src="src/android/HockeySDK-Android/HockeySDK-5.1.0.aar" />
```

Comment:
HockeySDK binary name is HockeySDK-Android-5.1.0.aar, so we should change either this line or binary name accordingly.

Reply:
I was thinking about this, too, because it's very counterintuitive when releasing. Let me update that.